doc/admin-guide/plugins/geoip_acl.en.rst

```diff
@@ -51,7 +51,7 @@ The three typical use cases are as follows:
    have something like ::
 
     map http://example.com/music http://music.example.com \
-      @plugin=geoip_acl.so @pparam=country @pparam=allow @pparam=US
+      @plugin=geoip_acl.so @pparam=country @pparam=allow @pparam=US \
```

Comment:
Am I reading this wrong? It looks like the next line is a new map?

Reply:
Nope, you are not! Good catch.